quickwit/quickwit-proto/build.rs

```diff
@@ -17,25 +17,16 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-fn main() -> Result<(), Box<dyn std::error::Error>> {
-    println!("cargo:rerun-if-changed=proto/ingest_api.proto");
```

Comment:
Those are actually already generated by `tonic::compile`.  You can see them when running `cargo` in verbose mode `-vv`.